Lesson27/ezgi/create/create-post.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+	<title>Create post</title>
+	<link rel="stylesheet" type="text/css" href="../styles.css" />
+</head>
+
+<body>
+	<div class="create-container">
+		<h1>Create post</h1>
+
+		<form action="" id="create-post">
+			<label for="title">Title:</label>
+			<textarea id="title" required></textarea>
+
+			<label for="body">Content</label>
+			<textarea id="body" required></textarea>
+			<button id="create-post-button" class="button form-button button--success" type="submit">Create
+				Post</button>
+		</form>
+	</div>
+	<a href="Lesson27/ezgi/index.html" class="button button--main" >main page</a>
```

Comment:
Correct path would be this:
```suggestion
	<a href="../index.html" class="button button--main" >main page</a>
```

You're using a "relative url" type of URL here. So the "address" is calculated starting in the current file (create-post.html). In relation to the current file, main page is located in the outer folder, that's why `../` are used before the file name. 

https://ahrefs.com/seo/glossary/relative-url